README.md

```diff
@@ -75,6 +81,8 @@ Six seconds is an eternity so you've been tasked with speeding up the code. Can
 
 #### Task 3. Reverse a Linked List
 
+## we did an exercise earlier this week reversing a SLL , yay!
```

Comment:
Yayy. Yeah, this is not an easy problem to figure out; it's good that you guys got some practice in!